grafana/dashboard/dev/streaming_actors_tokio.py

```diff
@@ -9,23 +9,23 @@ def _(outer_panels: Panels):
         outer_panels.row_collapsed(
             "Streaming Actors (Tokio)",
             [
-                panels.timeseries_actor_latency(
-                    "Actor Execution Time",
+                panels.timeseries_percentage(
+                    "Actor Execution Rate",
                     "",
                     [
                         panels.target(
-                            f"rate({metric('stream_actor_actor_execution_time')}[$__rate_interval]) > 0",
-                            "{{actor_id}}",
+                            f"rate({metric('stream_actor_execution_duration')}[$__rate_interval]) / 1000000000",
+                            "actor {{actor_id}} fragment {{fragment_id}}",
                         ),
                     ],
                 ),
-                panels.timeseries_actor_latency_small(
-                    "Tokio: Actor Fast Poll Time",
+                panels.timeseries_percentage(
+                    "Tokio: Actor Fast Poll Rate",
                     "",
                     [
                         panels.target(
-                            f"rate({metric('stream_actor_fast_poll_duration')}[$__rate_interval]) > 0",
-                            "{{actor_id}}",
+                            f"rate({metric('stream_actor_fast_poll_duration')}[$__rate_interval]) / 1000000000",
```

Comment:
We use an int counter instead for better performance, and consistency with other latency measures (e.g. backpressure metrics). We increment in `nanoseconds` to maintain precision, so we need to normalize by 1e9 here.